library/panic_unwind/src/emcc.rs

```diff
@@ -47,7 +47,12 @@ static EXCEPTION_TYPE_INFO: TypeInfo = TypeInfo {
     name: b"rust_panic\0".as_ptr(),
```

Comment:
Maybe change it to avoid unsoundness in combination with older rust versions? Same for the exception type for the other unwinding mechanisms.

Reply:
I think it's worth mentioning that it should be noted that catching a foreign unwind in Rust is UB. We are providing guards against this but it's not part of guarantee.

Currently there are no ways to hit this on stable, and if you hit this using nightly you are already in UB territory. So we *can* change this string (and the exception class), but we *don't have to*.